website/content/en/docs/add-ons/storage/efs/guide.md

```diff
@@ -312,7 +312,7 @@ kubectl logs -n $PVC_NAMESPACE image-classification-pvc-worker-0 -f
 ```
 
 ## 4.0 Cleanup
-This section cleans up the resources created in this guide. To clean up other resources, such as the Kubeflow deployment, see [Uninstall Kubeflow]({{< ref "/docs/deployment/uninstall-kubeflow.md" >}}).
```

Comment:
instead of removing, should we say

To, clean up other resources, see the uninstall sections according to the deployment option you used